pkg/assembler/clients/operations/vulMetadata.graphql

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Defines the GraphQL operations to ingest metadata for a vulnerability into GUAC
+
+mutation VulnHasMetadata(
+  $vulnerability: VulnerabilityInputSpec!, 
+  $score: VulnerabilityScoreInputSpec!, 
+  $vulnMetadata: VulnerabilityMetadataInputSpec!
+  ) {
+  ingestVulnerabilityMetadata(
+    vulnerability: $vulnerability, 
+    score: $score, 
+    vulnerabilityMetadata: $vulnMetadata
+  )
+}
+
+# Defines the GraphQL operations to bulk ingest metadata for a vulnerability into GUAC
```

Comment:
Nit: Let's make this comment `"` and attached (no empty line) to the schema. Same for the comment on line 18. This way it gets into the documentation.

Reply:
that seems to error @mihaimaruseac 
`operations/vulMetadata.graphql:18: invalid query-spec file operations/vulMetadata.graphql: Unexpected String "Defines the GraphQL operations to ingest metadata for a vulnerability into GUAC"`

Reply:
We can fix this with all the others in another PR.